<launch>
    <arg name="ego2lidar_x" default="-0.5"/>
    <arg name="ego2lidar_y" default="0.0"/>
    <arg name="ego2lidar_z" default="2.0"/>
    <node pkg="visualization" type="visualization" name="visualization">
        <param name="ego2lidar_x" value="$(arg ego2lidar_x)"/>
        <param name="ego2lidar_y" value="$(arg ego2lidar_y)"/>
        <param name="ego2lidar_z" value="$(arg ego2lidar_z)"/>
    </node>  
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find visualization)/resources/hmi.rviz" />
</launch>